src/components/secondPassphraseInput/secondPassphraseInput.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Input from 'react-toolbox/lib/input';
+import { isValidPassphrase } from '../../utils/passphrase';
+
+class SecondPassphraseInput extends React.Component {
+  componentDidMount() {
+    if (this.props.hasSecondPassphrase) {
+      this.props.onError(undefined, '');
+    }
+  }
+
+  handleChange(value) {
+    this.props.onChange(value);
+    const error = this.validate(value);
+    if (error) {
+      this.props.onError(error, value);
+    }
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  validate(value) {
+    if (!value) {
+      return 'Required';
+    } else if (!isValidPassphrase(value)) {
+      return 'Invalid passphrase';
+    }
+    return undefined;
```

Comment:
I know it's redundant. But AFAIK that's the whole point of `consistent-return` eslint rule. To improve readability through redundancy. I think it's also more readable than using a variable and return once.